Post/tests.py

```diff
@@ -1,13 +1,44 @@
 from Post.models import Post
+from django.db.models.query import QuerySet
+import pytest
 
 
-class TestPost():
+@pytest.mark.django_db
+def test_post_str(post):
 
-    def test_post_creation(self):
+    assert str(post) == "Shoval traveled The Dead Sea and wrote: beautiful place"
 
-        post1 = Post(nameOfPoster='Shoval',
-                     nameOfLocation='The Dead Sea',
-                     photoURL='www.testPost.com',
-                     Description='beautiful place')
 
-        assert str(post1) == "Shoval traveled The Dead Sea and wrote: beautiful place"
+@pytest.mark.django_db
+def test_post_creation(post):
+    post.save()
```

Comment:
Where do you check post content?

Reply:
@Yarboa, Do you mean to add "assert list(posts.values_list('nameOfPoster',....)== [ (..) ,..., (..) ]"  ?
to check if the content of the posts is equal to what we intended it to be?

If so, I tried to do that, but we have lots of posts in the DB, and it will be very long.
also, I think that if we add a new post we will need to change the test. 

I will be happy to hear a suggestion on how to do that!

Reply:
I would at least check the following:
You are using fixture post
In this test: 
test_post_creation 
I would check that that the following exist in DB after read, not just the ID
return Post(nameOfPoster='Shoval',
                nameOfLocation='The Dead Sea',
                photoURL='www.testPost.com',
                Description='beautiful place')

Reply:
Hi @Yarboa,
I'm sorry, I thought you were talking about 'test_all_posts'.

In this test I wrote  "Post.objects.get(pk=post.id) == post "
and I think it is checking the content because I get the post from the DB (by getting the post with the correct ID), and then I'm checking if it is equal to 'post'. is it enough or should I do something else?

Reply:
sorry, i missed this part